prowler/providers/github/services/repository/repository_inactive_not_archived/repository_inactive_not_archived.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "github",
+  "CheckID": "repository_inactive_not_archived",
+  "CheckTitle": "Check for inactive repositories that are not archived",
+  "CheckType": [],
+  "ServiceName": "repository",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "medium",
+  "ResourceType": "GitHubRepository",
+  "Description": "Ensure that repositories with no activity in the last 6 months are reviewed and considered for archival. Inactive repositories may have outdated dependencies or security configurations that could pose security risks.",
```

Comment:
Let's make it configurable so we don't have to hardcode here 6 months.

Reply:
Sure!